linux/VSTPlugin-linux.cpp

```diff
@@ -23,20 +23,49 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 AEffect* VSTPlugin::loadEffect() {
 	AEffect *plugin = nullptr;
 
-	wchar_t *wpath;
-	os_utf8_to_wcs_ptr(pluginPath.c_str(), 0, &wpath);
-	soHandle = LoadLibraryW(wpath);
+	// Why is this a (non-portable) wide char?
```

Comment:
Also my next PR will include formatting fixes (if you haven't already done it) for everything.